tests/csm/rest/test_capacity_quota.py

```diff
@@ -641,3 +646,354 @@ def test_40640(self):
+                                              self.akey, self.skey, object_size=random_size)
+            assert resp, "Put object Failed"
+        self.log.info("Step 2: Perform GET API to get capacity usage stats using - manage")
+        resp = self.csm_obj.get_capacity_usage("user", self.user_id)
+        assert resp.status_code == HTTPStatus.OK, \
+            "Status code check failed for get capacity"
+        self.log.info("Step 3: Verify above count matches aws response")
+        status, output = run_local_cmd(commands.CMD_AWSCLI_READABLE, chk_stderr=True)
+        assert status, "list objects operation failed"
+        total_objects, total_size = self.csm_obj.get_objects_and_size(output)
+        t_obj = resp.json()["capacity"]["s3"]["user"][0]["objects"]
+        t_size = resp.json()["capacity"]["s3"]["user"][0]["used"]
+        assert_utils.assert_equals(total_objects, t_obj, "Number of objects not equal")
+        assert_utils.assert_equal(total_size, t_size, "Total Size mismatch found")
+        self.log.info("##### Test ended -  %s #####", test_case_name)
+
+    @pytest.mark.skip("Feature not ready")
+    @pytest.mark.lc
+    @pytest.mark.csmrest
+    @pytest.mark.cluster_user_ops
+    @pytest.mark.parallel
+    @pytest.mark.tags('TEST-41750')
+    def test_41750(self):
+        """
+        Test GET capacity stats for create IAM user and Put objects using monitor users
+        """
+        test_case_name = cortxlogging.get_frame()
+        self.log.info("##### Test started -  %s #####", test_case_name)
+        test_cfg = self.csm_conf["test_41750"]
+        max_size = test_cfg["max_size"]
+        random_size = self.cryptogen.randrange(1, max_size)
+        num_objects = math.floor(max_size / random_size)
+        self.log.info("Step 1: Create N objects of Random size totals to S bytes")
+        for _ in range(0, num_objects):
+            resp = s3_misc.create_put_objects(self.obj_name, self.bucket,
+                                              self.akey, self.skey, object_size=random_size)
+            assert resp, "Put object Failed"
+        self.log.info("Step 2: Perform GET API to get capacity usage stats using - monitor")
+        resp = self.csm_obj.get_capacity_usage("user", self.user_id)
+        assert resp.status_code == HTTPStatus.OK, \
+            "Status code check failed for get capacity"
+        self.log.info("Step 3: Verify above count matches aws response")
+        status, output = run_local_cmd(commands.CMD_AWSCLI_READABLE, chk_stderr=True)
+        assert status, "list objects operation failed"
+        total_objects, total_size = self.csm_obj.get_objects_and_size(output)
+        t_obj = resp.json()["capacity"]["s3"]["user"][0]["objects"]
+        t_size = resp.json()["capacity"]["s3"]["user"][0]["used"]
+        assert_utils.assert_equals(total_objects, t_obj, "Number of objects not equal")
+        assert_utils.assert_equal(total_size, t_size, "Total Size mismatch found")
```

Comment:
verify UID also. resp.json()["capacity"]["s3"]["user"]["id"]
Verify used_total is greater than used

Reply:
added verifications